### PR TITLE
Completed Makefile functionality

### PR DESCRIPTION
**Pull request by mpetersoCode55**

### CODE REVIEW

File: script/makefile-tools/convertStringToByte32.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+  #!/bin/bash
```
Comment (excalq) [inline script/makefile-tools/convertStringToByte32.sh:1]: Why not require a single arg `INPUT=${1:-'Test'}`, rather than using `-s ARG`?

Comment (excalq) [reply script/makefile-tools/convertStringToByte32.sh:1]: (Which would remove the need for lines 5-22).

**Review by excalq — APPROVED** [review]
👍 Good stuff! As noted, I think the shell script's arg handling could be simplified.

File: script/makefile-tools/convertStringToByte32.sh (hunk 1 above)
Comment (mpetersoCode55) [reply script/makefile-tools/convertStringToByte32.sh:1]: Good catch! Implemented

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# (-include to ignore error if it does not exist)
+-include .env
+
+###############################################
+# Cast Sends - Setters
+###############################################
+
+# Create a Account Min Max Token Balance rule. To use a specific string for the tag use the following syntax:
+# make setRule INPUT=tag
+# otherwise if INOUT is not specified, "Test" will be used for the tag.
```
Comment (ShaneDuncan602) [inline Makefile]: INOUT

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+# Output is parsed and the created rule id is given as output to the make call
+createRule: createRuleCastSend
+	@echo $(shell sh ./script/makefile-tools/parseRuleID.sh)
```
Comment (ShaneDuncan602) [inline Makefile:14]: This is great. Extremely helpful for clients because wading through that output sucks.

**Review by ShaneDuncan602 — APPROVED** [review]
This is great. I plan to use this often.

**Review by TJ-Everett — APPROVED** [review]
Approved for me. This is great work!

**Pull request merged**